app/tests/test_quizzes.py

```diff
@@ -9,10 +9,13 @@ def setUp(self):
         self.id = self.quiz["_id"]
         self.length = len(self.quiz_data["question_sets"][0]["questions"])
 
+    def test_setup_quizId_and_quiz(self):
```

Comment:
the name of this test method seems a little off.
Where are you setting up quiz id and quiz?

This is just verifying that the id returned from create quiz call is the same as the id of the quiz returned from get quiz call, right?
Can you try renaming it based on that?